internal/host/validator.go

```diff
@@ -760,6 +768,10 @@ func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationConte
 		return ValidationSuccess
 	}
 
+	if len(c.host.Inventory) == 0 {
```

Comment:
I am not sure we should say the validation is pending just because we cannot print the hostname in the validation message.  You can use the id instead.

Reply:
Ok, I'll use the ID then. Ignore my previous comment then :smile:

Reply:
Removed this conditional. Using the host.ID removes the need for the inventory availability.